src/Text/Pretty/Simple.hs

```diff
@@ -734,5 +734,8 @@ layoutStringAnsi opts = fmap convertStyle . layoutString opts
 -- >>> pPrint "this string has non-printable characters: \x8 and \x9"
 -- "this string has non-printable characters: \x8 and \x9"
 --
+-- >>> pPrint 'λ'
+-- 'λ'
+--
```

Comment:
This doesn't flow brilliantly because the existing documentation is misleading - non-printable characters aren't the only ones which get escaped.

Reply:
I'm not quite sure what you're referring to here.  The changes that you're making here seem like they make sense.

Is there some easy change we can make to the pretty-simple documentation to make it clearer what is going on?